nav2_costmap_2d/test/unit/keepout_filter_test.cpp

```diff
@@ -34,9 +34,9 @@
 
 using namespace std::chrono_literals;
 
-static const std::string FILTER_NAME = "keepout_filter";
-static const std::string INFO_TOPIC = "costmap_filter_info";
-static const std::string MASK_TOPIC = "mask";
+static const char FILTER_NAME[]{"keepout_filter"};
```

Comment:
Why not leave as string, you just cast to string later

Reply:
Hmm.. 

`Static/global string variables are not permitted.`

Let me check for alternates